test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -687,9 +753,17 @@ UCS_TEST_P(test_uct_cm_sockaddr, many_clients_to_one_server)
     for (i = 0; i < num_clients; ++i) {
         client_test = uct_test::create_entity();
         m_entities.push_back(client_test);
-        client_test->max_conn_priv = client_test->cm_attr().max_conn_priv;
+
+        client_test->index_m_entities = get_entity_index(client_test);
+        EXPECT_GT(client_test->index_m_entities, -1);
```

Comment:
`EXPECT_LE(0, client_test->index_m_entities);`
expected value goes first

Reply:
i used GT and expect the result, which goes first, to be greater than -1.
why is LE better?

Reply:
agree, there is no difference for GT and LE